src/js/stores/AuthStore.js

```diff
@@ -96,6 +96,20 @@ class AuthStore extends GetSetBaseStore {
     }
   }
 
+  getUserLabel() {
+    const user = this.getUser();
+
```

Comment:
This is nit-picking so no need to change.. but in some cases when declaring a variable there is no need to break to a new line.

Reply:
:angry: